matminer/featurizers/tests/test_structure.py

```diff
@@ -304,6 +305,23 @@ def test_op_structure_fingerprint(self):
         for i in range(52, len(opvals)):
             self.assertAlmostEqual(int(opvals[i] * 500 + 0.5), 0)
 
+    def test_ewald(self):
+        # Add oxidation states to all of the structures
+        for s in [self.nacl, self.cscl, self.diamond]:
+            s.add_oxidation_state_by_guess()
+
+        # Test basic
+        ewald = EwaldEnergy(accuracy=2)
+        self.assertArrayAlmostEqual(ewald.featurize(self.diamond), [0])
+        self.assertLess(ewald.featurize(self.nacl)[0], 0)
+        self.assertLess(ewald.featurize(self.nacl),
+                        ewald.featurize(self.cscl))  # Atoms are closer in NaCl
+
+        # Perform Ewald summation by "hand",
+        #  assuming that EwaldSummation from pymatgen is correct
+        self.assertArrayAlmostEqual([EwaldSummation(self.nacl, acc_factor=2).total_energy],
+                                    ewald.featurize(self.nacl))
+
     def tearDown(self):
```

Comment:
this doesn't seem to be your code, but if this tearDown() is not needed you can just remove it